core/station/impl/src/repositories/request.rs

```diff
@@ -387,6 +388,19 @@ impl RequestRepository {
         Ok(entries.into_iter().map(|(id, _)| id).collect())
     }
 
+    pub async fn cancel_request(
```

Comment:
nit: potentially async could be removed here

Reply:
Good point! Done.